ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -516,14 +521,15 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				DEVELOPMENT_TEAM = JVG2N36PSN;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = Runner/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.telnyx.telnyxFlutterWebrtc;
+				PRODUCT_BUNDLE_IDENTIFIER = com.isaac.telnyxFlutterWebrtc;
```

Comment:
is that correct?

Reply:
Yeah, I still don't have access to com.telnyx.telnyxFlutterWebrtc gor Firebase. I had to use my own. I'll just make the correction . I'll have to change it to mine everytime I want to test the app

Reply:
It's generated code as well